Update qb-customs.md

12LetterMeme opened the pull request:
Updating the restrictions section to include how to whitelist jobs and specific vehicle classes. Having the full range of options shown would be beneficial in the docs, maybe just change my wording a little better so people can maximize the potential uses. 

For example

with allowed classes, better wording for that so people don't think they need to put all vehicle classes in there and it's more just for allowing specific vehicle classes like 18(emergency). just so they understand there can either have all vehicles allowed or jut specify which classes in a shop. The same goes for restricted

amantu-qbit reviewed (approved):
Just speeling correction

12LetterMeme committed:
Update qb-customs.md

Updating the restrictions section to include how to whitelist jobs and specific vehicle classes!
